core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -37,7 +37,7 @@ import scala.collection.{Iterable, immutable, mutable}
 private[log] sealed trait LogCleaningState
 private[log] case object LogCleaningInProgress extends LogCleaningState
 private[log] case object LogCleaningAborted extends LogCleaningState
-private[log] case object LogCleaningPaused extends LogCleaningState
+private[log] case class LogCleaningPaused(times: Int) extends LogCleaningState
```

Comment:
Would it be more readable to replace `times` with `count`?

Now that we are making the state more complicated, can we add comment to explain transition rules for these states, similar to the comment in `ReplicaStateMachine.scala` and `PartitionStateMachine.scala`?

Reply:
I will add comments about transition rules.

Reply:
`pausedCount` sounds good.